domains/dhilipan.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "MysticalPvE",
+           "email": "",
+           "discord": "1229782675678826496",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.MlTeWHEABWqoPSloZ0X-_FTt5rjNC07O8ftIk922HgB_eMfLDyy4Ke5ejBdtrdBeIw-IF_U3nV3GiNuJ3rHtDgBu3YlHBpu85qGqDpi7Xy8Wo0C7LO9YiVyQ7YQw3Yw2uJ3xCbpOD6KWZWvpJjzruiTxiIf5jYYPe9qFNxudIn4RULpMm2O4Bosg3FWNLmirfrFxtnOHV_iKjQxcZ8BpS0F_p7q5hbLK_kiiGbEwZM6hT6W6D5stbkYkcUW0Lmge2Yig9SJmusLezfOjcdy49Zj0xdolJz_QYRgp8kNRTPCqUvA43ZjP97-3EKhw5mHp1issH4x7YB1C8oPeczuzmg.ayCeTs0MbCBOn7xbcYpFFg.rHuNuNUH6p5fIvxcV-gMSm3mbSoS6_eX4itnTawpdJgbzP_Xk79OjXWGXoxJvN33nyGUU_tSsXsk8_rvzzf6kFPBiZ6XBVWOw3hgV1GacwI.Cu-_mzO33_Eww6J-AfxsNw"
+        },
+    
+        "record": {
+            "A": ["44.217.161.11"]
```

Comment:
I cannot reach your website, 404

Reply:
apparently netlify doesn't work A records (it's all connected with load balancing servers or smth). I'll create a new one with CNAME records.